README.md

```diff
@@ -214,6 +214,8 @@ This should prevent the provider from asking you for a Datadog API Key and allow
 
 ### Monitoring IAM Activity
 
+> NOTE: From Control Tower 3.0 and later, this is only performed within the master account.
```

Comment:
Can you add this to the Upgrading.md and reword a bit?

From Control Tower 3.0 and later, an organization-level AWS CloudTrail is deployed and the CloudWatch log group `aws-controltower/CloudTrailLogs` is not used anymore. During upgrading these log groups will be removed.